tests/jballerina-unit-test/src/test/resources/test-src/expressions/varref/record-variable-reference.bal

```diff
@@ -369,6 +369,69 @@ function testMappingBindingWithSingleNameFieldBinding() {
     assertEquality("bar4", b);
 }
 
+function testMappingBindingPatternsAsMemberBindingPatterns() {
```

Comment:
I would also include tests with open records with an explicit rest descriptor.

Reply:
Do you mean something like this?
```ballerina
[record {int x; int y;}] [{...x}] = [{x: 1, y: 2}];
```

Reply:
No. 

> open records with an explicit rest descriptor

```ballerina
record {| int x; int y; anydata...; |}
```